charts/castai-kvisor/values.yaml

```diff
@@ -145,7 +145,7 @@ securityContext: {}
 agentContainerSecurityContext:
   readOnlyRootFilesystem: true
   runAsNonRoot: true
-  runAsUser: 1004
```

Comment:
Not sure how this changes anything, especially, when > 1000 ids belong to non root users.

Reply:
65532 is the ID of nonroot user in distroless images